v3.3/glfw/window.go

```diff
@@ -201,6 +201,12 @@ func GoWindow(window unsafe.Pointer) *Window {
 	return &Window{data: (*C.GLFWwindow)(window)}
 }
 
+// GLFWWindow returns a *C.GLFWwindow reference (i.e. the GLFW window itself). This can be used for
```

Comment:
There is already `.Handle()` for this purpose.
https://github.com/go-gl/glfw/blob/6f7a984d4dc470c3f197229ad1991ae9e211bba2/v3.3/glfw/window.go#L191-L196